web/README.md

```diff
@@ -112,9 +112,38 @@ run a demo locally with `npm run demo`, or [install the extension in your browse
 
 ### Testing
 
-To run all of the tests in this project, we currently require that you have [Chrome installed to its default location](https://www.google.com/chrome/).
+There are two parts of tests to this project:
+- Regular node tests, ran through `npm run test`. You must have built everything first as above. These have no special requirements.
+- Browser based tests, ran through `npm run wdio` with extra arguments as below. These take longer to run and require some setup.
 
-First, ensure you've build every package (see above), and then run `npm run test` to run the full suite of tests.
+## Browser based tests
+There are full integration tests that require a browser to run. We don't make any assumptions about your environment, and so you must specify it yourself.
+
+To run these tests, first build the project as above, then use `npm run wdio -- --arg1 --arg2` etc.
+
+### Local Browsers
```

Comment:
Okay, maybe this is the exception for being in title case...? Either way, it's inconsistent.